tools/dragen-transcriptome/3.9.3/dragen-transcriptome__3.9.3.cwl

```diff
@@ -353,6 +353,61 @@ inputs:
       Specify the name of the rRNA sequences to use for filtering.
     inputBinding:
       prefix: "--rrna-filter-contig"
+  read_trimmers:
+    label: read trimming
+    type: string?
+    doc: |
+      To enable trimming filters in hard-trimming mode, set to a comma-separated list of the trimmer tools 
+      you would like to use. To disable trimming, set to none. During mapping, artifacts are removed from all reads.
+      Read trimming is disabled by default.
+    inputBinding:
+      prefix: "--read-trimmers"
```

Comment:
inputBinding should be of the format:

```
prefix: "--parameter-key="
separate: False
```

Reply:
We were still using older inputBinding option for v3.9. I have updated all inputs to use consistent bindings.

Reply:
https://github.com/umccr/cwl-ica/pull/391/commits/8c69f8334868ef36364852786fc37e3e1f8cdb22